tests/resources/dynamic-vars.py

```diff
@@ -31,7 +31,7 @@ def getEnvironment():
         return TestEnvironment.LOCAL
 
 def getName(image):
-    return {TestEnvironment.DRONE: 'harbor.ci.drone.local/library/{}'.format(image),
+        return {TestEnvironment.DRONE: 'wdc-harbor-ci.eng.vmware.com/default-project/{}'.format(image),
```

Comment:
same comment about the registry being set via environment variable, but more compelling now it's being embedded in code in multiple places that have to match.